public/script.js

```diff
@@ -292,11 +307,18 @@ function clearDataOnClick() {
   }
 }
 
-function copyToClipBoard() {
+function copyLinkToClipBoard() {
```

Comment:
- copyLinkToClipBoard and copySecretToClipBoard has duplicated code
- I thought I would be able to pass in the element id as a parameter (ex: pass in "secret-message-box")
but that does not work

Reply:
feel free to refactor it as part of a separate PR, but it's not crucial.